src/OpenRealEstate.Core/Rental/RentalListing.cs

```diff
@@ -1,4 +1,4 @@
-﻿using System;
+using System;

```

Comment:
Prolly can revert all the changes in this file.

Reply:
done